cf-usb/cf-usb.yaml

```diff
@@ -136,7 +156,9 @@ jobs:
     - put: docker.mysql
       params:
         build: mysql-binary
-        load_base: docker.base-fixed
+        load_bases:
+        - docker.base-fixed
+        - docker.buildbase-fixed
         tag_as_latest: true
         tag: mysql-binary/tag
```

Comment:
The tag doesn't seem to match what we actually put in the helm config; I can't pull it manually.

Reply:
Should be fixed with commit 293ec24c790b75cc444f7d32a793d593f16d8577